packages/nuxt/src/core/features.ts

```diff
@@ -0,0 +1,35 @@
+import { addDependency } from 'nypm'
+import { isPackageExists } from 'local-pkg'
+import { logger } from '@nuxt/kit'
+import prompts from 'prompts'
+
+export async function ensurePackageInstalled (rootDir: string, name: string) {
```

Comment:
PR welcome to move all of logic from this util directly to nypm. We could early integrate with consola v3 there too also removing `prompts` install overhead